hack/dev/skaffold.Dockerfile

```diff
@@ -2,7 +2,6 @@ FROM kotsadm:cache AS builder
 
 ENV PROJECTPATH=/go/src/github.com/replicatedhq/kots
 WORKDIR $PROJECTPATH
-RUN mkdir -p web/dist && touch web/dist/README.md
```

Comment:
are these not needed anymore? I thought this ensured the web stuff worked

Reply:
This is for the dev env and web is served from the kotsadm-web service not the dist directory.